src/test/java/pl/javastart/streamstask/ExampleTest.java

```diff
@@ -1,6 +1,135 @@
 package pl.javastart.streamstask;
 
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.math.BigDecimal;
+import java.util.*;
+import java.util.stream.Collectors;
+
 public class ExampleTest {
 
+    @BeforeEach
+    public List<User> takeUserList() {
+        List<User> users = new ArrayList<>();
+
+        users.add(new User(1L, "Alicja", 20));
+        users.add(new User(2L, "Dominik", 15));
+        users.add(new User(3L, "Patrycja", 25));
+        users.add(new User(4L, "Marcin", 30));
+        users.add(new User(5L, "Tomek", 18));
+        users.add(new User(6L, "Damian", 26));
+
+        return users;
+    }
+
+    @BeforeEach
+    public List<Expense> takeExpenseList() {
+        List<Expense> expenses = new ArrayList<>();
+
+        expenses.add(new Expense(1L, "Buty", new BigDecimal("149.99"), ExpenseType.WEAR));
+        expenses.add(new Expense(1L, "Sałatka", new BigDecimal("14.99"), ExpenseType.FOOD));
+        expenses.add(new Expense(2L, "Bluza", new BigDecimal("100"), ExpenseType.WEAR));
+        expenses.add(new Expense(2L, "Skarpetki", new BigDecimal("39"), ExpenseType.WEAR));
+        expenses.add(new Expense(2L, "Pizza", new BigDecimal("25"), ExpenseType.FOOD));
+
+        return expenses;
+    }
+
+    @Test
+    public void shouldReturnUsersWithEndsA() {
```

Comment:
To, że imiona kobiet kończą się na A to tylko detal techniczny. W tej metodzie sprawdzasz, czy zwrócona jest kolekcja kobiet i to powinna oddawać nazwa